_includes/devdoc/guide_dash_features.md

```diff
@@ -109,6 +109,100 @@ had already been confirmed to a block depth of 5 in the blockchain.
+
+There are still some limitations on LLMQ-based InstantSend transactions:
+
+* Transaction inputs must either:
+  * Be in a block that has a ChainLock
+  * Have at least the number confirmations (block depth) indicated by the table below
+
+  | **Network** | **Confirmations Required** |
+  |---------|--------------|
+  | Mainnet | 6 Blocks |
+  | Testnet | 2 Blocks |
+  | Regtest | 2 Blocks |
+  | Devnet  | 2 Blocks |
+
+
+Improvements from the old InstantSend system:
+
+* Changed: Transactions can be chained if the inputs are from transactions that are also locked
+* Changed: InstantSend locks are attempted for all transactions (`tx` messages) - no need to request it via the special message (`ix` message)
+* Changed: Only need to receive a single `islock` message - no need to track votes (`txlvote` messages) for each input
+* Removed: Limit on number of inputs
+* Removed: Limit on transaction value
+* Removed: Timeout for lock - transaction locks will only be removed once the transaction is confirmed in a ChainLocked block
+* Removed: Custom InstantSend fee
+
+Note: A transaction will __not__ be included in the block template (from `getblocktemplate`) unless it:
+
+ 1. Has been locked, or
+ 2. Has been in the mempool for >=10 minutes (`WAIT_FOR_ISLOCK_TIMEOUT`)
+
+A miner may still include any transaction, but blocks containing only locked
+transactions (or ones older than the timeout) should achieve a ChainLock faster.
+This is desirable to miners since it prevents any reorgs that might orphan their
+block.
+
+*InstantSend Data Flow*
+
+| **InstantSend Client** | **Direction**  | **Peers**   | **Description** |
+| `tx` message                | → |                         | Client sends InstantSend transaction
+| **LLMQ Signing Sessions**   |   |                         | Quorums internally process locking |
+|                             |   |                         | Quorum(s) responsible for the transaction's inputs lock the inputs via LLMQ signing sessions
+|                             |   |                         | Once all inputs are locked, the quorum responsible for the overall transaction creates the transaction lock (`islock`) via an LLMQ signing session
+| **LLMQ Results**             |   |                         | Quorum results broadcast to the network |
+|                             | ← | `inv` message (islock)  | Quorum responds with lock inventory
+| `getdata` message (islock)  | → |                         | Client requests lock message
+|                             | ← | `islock` message        | Quorum responds with lock message
+
+{% endautocrossref %}
+
+### ChainLocks
+{% include helpers/subhead-links.md %}
+
+{% autocrossref %}
+
+Dash's ChainLock feature leverages [LLMQ Signing Requests/Sessions](#llmq-signing-session)
+to reduce uncertainty when receiving funds and remove the possibility of 51%
+mining attacks.
+
+For each block, an LLMQ of a few hundred masternodes (300-400) is selected and each
```

Comment:
I thought it is exactly 400 MNs

Reply:
That's the ideal case where there aren't any bad/non-participating members in the DKG setup. It can go as low as 300 though.
https://github.com/dashpay/dash/blob/develop/src/chainparams.cpp#L149-L151
```
        .name = "llmq_400_60",
        .size = 400,
        .minSize = 300,
        .threshold = 240,
```

Reply:
Even if some MNs are bad/non participating, aren't they technically in the LLMQ?

Reply:
They would only be in the LLMQ if they participated in DKG. If they don't participate in DKG, they have no contribution to the quorum's key (and therefore aren't part of the quorum). 
@codablock, Can you confirm what I said is correct?

Reply:
Ahh okay, that seems to make some sense. And if the minimum song participate in DKG then the LLMQ is invalid/not used etc

Reply:
Yes, if there are less than 300 good members (for `llmq_400_60`), the "candidate quorum" will not be able to form a key and thus not really exist.

Reply:
Confirming what @thephez wrote.